src/workerd/server/server.c++

```diff
@@ -964,6 +964,27 @@ kj::Own<Server::Service> Server::makeDiskDirectoryService(
 
 // =======================================================================================
 
+class Server::InspectorServiceIsolateRegistrar final {
+public:
+  InspectorServiceIsolateRegistrar() {}
+  ~InspectorServiceIsolateRegistrar();
```

Comment:
KJ style says all destructors should be declared either `noexcept(false)`, or if noexcept behavior is explicitly desired, then `noexcept`.

(I hate that C++11 changed the default.)

Reply:
Thanks, done